developer-docs-site/docs/guides/up-and-running.md

```diff
@@ -1,10 +1,9 @@
 ---
-title: "Getting Started"
-slug: "getting-started"
-sidebar_position: 0
+title: "Up and Running"
+slug: "up-and-running"
```

Comment:
I want to believe, but "up and running" seems more ambiguous than "getting started"

Reply:
Okay, I was sort of on the fence with it, but now you pushed me over. i will change it.